### ISSUE TITLE
No impact from changing hot water loop max temp to 0C. (CR #818)

### ISSUE THREAD

**Issue by axelstudios** [NREL/EnergyPlus#24]
## 
#### Description

MJW 99.12.03

Test 3251 is the StarTeam VAVSimple input file 
with the Hot Water Loop Max Temperature set 
from 100C back to 0C.  There appears to be no 
change in results.  Test was done for 10 days 
in Jan, Apr and Jul using dry/cool weather.  
Used 12-1-99 version of exe.

Plant configuration shows heating coming from 
purchased heating.  No hot water pump is 
specified.  Must just be getting Btus (or Watts) 
from purchased heating.

Is this why changing hot water loop max temp has no 
effect?  But shouldn't there be a loop temp associated with purchased heating anyway?

Input file:  065-VAVSimple-3251.idf
Weather:  drycold.wea
## 
#### Fix

This defect has been allowed to age, but it won't go away. Beta2
code still shows the same noneffect - lowering the hot water loop
maximum temperature in TermReheat.idf to 11 deg C has no effect.

The loop max Temperature input IS transferred to the loop nodes.
However the purchased hot water component never looks at the
incoming or outgoing node max temperature. All it does is add enough
heat to bring the water temperature up to the set point temperature.
In other words, it looks at the set point temperature, but not the max
temperature. It doesn't worry whether the max is less than the set point.

I suspect that the same holds true of some of the other simple "BTU"
component models. This is really a design issue not a defect. How
are we going to treat the max and min loop temperatures in our
component models? I'll mark this defect "as designed" and add it
to the list of things to discuss in Golden, May 2000.

Fred Buhl 3/31/2000
## 

External Ref: 65
Last build tested: `00.03.29 Build 014, Beta 2`
